### PR TITLE
skip Options serialize in Cpu, svd2yaml example

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: svd-parser/examples/svd2yaml.rs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    File::open(&svd_fn)
+        .expect("Failed to open SVD input file")
+        .read_to_string(svd_xml)
+        .expect("Failed to read SVD input file to a String");
+
+    // Use the 'svd_parser' crate to parse the file.
+    let device = svd::parse(svd_xml).expect("Failed to parse the SVD file into Rust structs");
+
+    // Convert the parsed data into YAML format.
+    let v: Value = yaml::to_value(device).expect("Failed to parse Rust structs into YAML format");
```
Comment (thejpster) [inline svd-parser/examples/svd2yaml.rs:31]: I think parse is the wrong word here - maybe 'serialise' is better

**Review by thejpster — DISMISSED** [review]
I had a couple of suggestions for the example. The Option changes seem fine

**Comment by burrbull** [comment]
Is that better?

**Review by Emilgardis — APPROVED** [review]

**Comment by burrbull** [comment]
bors r=Emilgardis

**Pull request merged**